tests/integration_tests/test_ingest_component_pathfinders.py

```diff
@@ -505,3 +509,66 @@ def test_ingest_pf_r1_general_and_cross_table_validation_errors(
         },
     ]
     assert validation_errors == expected_validation_errors
+
+
+def test_ingest_pf_r1_file_success_2(test_client_reset, pathfinders_round_1_file_success, test_buckets):
+    """Tests that submitting_account_id and submitting_user_email values are saved to
+    Submission model successfully."""
+
+    endpoint = "/ingest"
+    response = test_client_reset.post(
+        endpoint,
+        data={
+            "excel_file": pathfinders_round_1_file_success,
+            "fund_name": "Pathfinders",
+            "reporting_round": 1,
+            "auth": json.dumps(
+                {
+                    "Programme": [
+                        "Bolton Council",
+                    ],
+                    "Fund Types": [
+                        "Pathfinders",
+                    ],
+                }
+            ),
+            "do_load": True,
+            "submitting_account_id": "0000-1111-2222-3333-4444",
+            "submitting_user_email": "testuser@levellingup.gov.uk",
+        },
+    )
+
+    assert response.status_code == 200, f"{response.json}"
+
+    sub = Submission.query.filter_by(submission_id="S-PF-R01-1").one()
+    assert sub.submitting_account_id == "0000-1111-2222-3333-4444"
+    assert sub.submitting_user_email == "testuser@levellingup.gov.uk"
+
+
+def test_save_submission_file_name_and_user_metadata():
```

Comment:
this needs to use a test fixture that resets the changes, made within the test, otherwise there's a big risk of the data leaking into another test (probably in a separate module, or if someone extends this one). I think in this case the changes are not committed to the DB (from what I can tell by quickly reading it) so the appropriate fixture is probably seeded_test_client_rollback